### PR TITLE
Added documentation for Artifactory Package

### PR DESCRIPTION
**Pull request by jainishshah17**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joerg84) [inline .idea/vcs.xml]: @jainishshah17 is this supposed to be in here?

**Comment by jainishshah17** [comment]
@joerg84 Removed

**Comment by joerg84** [comment]
LGTM, @ssk2 could you have a short look?

**Comment by ssk2** [comment]
These screenshots are from DC/OS 1.9. We should retarget this for 1.9/?

**Comment by jainishshah17** [comment]
@ssk2  can you give me access to any DC/OS 1.8 env I can retake screenshot.

**Comment by ssk2** [comment]
@jainishshah17: we should have both ideally. Do we need screenshots? @joerg84 what is normal for the other examples? I know the docs team likes to stay away from them for this reason (they need frequent updating).

**Comment by jainishshah17** [comment]
@ssk2 Do you want me to remove images and add CLI example?

**Comment by ssk2** [comment]
@jainishshah17: that's probably a good idea. It'll be easier to deal with the future.

**Comment by ssk2** [comment]
@colin-mesosphere will be reviewing this too.

**Comment by jainishshah17** [comment]
@ssk2 Made changes please review it.

**Comment by joerg84** [comment]
@ssk2 @colin-msphere PTAL

**Comment by joerg84** [comment]
@ssk2 @colin-msphere PTAL

**Comment by ryadav88** [comment]
@ssk2 @colin-msphere PTAL. 
We just got another PR for artifactory package and I want to make sure this example gets merged along with the Universe PR https://github.com/mesosphere/universe/pull/1007

**Comment by ssk2** [comment]
Taking this PR over by opening a new one.

**Pull request closed without merging**